src/nemo_run/core/runners/fdl_runner.py

```diff
@@ -1,72 +1,50 @@
 # SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-# http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
 
-import os
-from pathlib import Path
-import fiddle as fdl
-from nemo_run.core.serialization.zlib_json import ZlibJSONSerializer
 import typer
 
-from nemo_run.config import Partial
-from nemo_run.core.packaging.base import Packager
-from nemo_run.run.task import dryrun_fn
+fdl_runner_app = typer.Typer(pretty_exceptions_enable=False)
 
-app = typer.Typer(pretty_exceptions_enable=False)
```

Comment:
I would prefer leaving this file unchanged if the changes are not mandatory. This is a runtime file so any changes to this file are likely to break backwards compatibility.

Reply:
It's needed to make reduce the chance significantly that there are name-conflicts between the two `__main__`s. I would consider this as internal API, since we just trigger this inside. All the args etc are all the same so I don't think it will break anything.